src/binding/petsc4py/src/petsc4py/PETSc/Viewer.pyx

```diff
@@ -94,13 +111,33 @@ cdef class Viewer(Object):
         self.obj = <PetscObject*> &self.vwr
         self.vwr = NULL
 
-    def __call__(self, Object obj):
+    def __call__(self, Object obj) -> None:
+        """View a generic object."""
```

Comment:
This is fine but won't get rendered anywhere. For other objects like this, we added a note in the class docstring with an example. I have added this as a suggestion.